src/printer.ts

```diff
@@ -582,11 +581,14 @@ export class PugPrinter {
 		{ trimTrailingSemicolon = false }: FormatDelegatePrettierOptions = {}
 	): string {
 		val = val.trim();
+		const options: Options = { ...this.codeInterpolationOptions };
 		const wasQuoted: boolean = isQuoted(val);
 		if (wasQuoted) {
+			options.singleQuote = !this.options.pugSingleQuote;
 			val = val.slice(1, -1); // Remove quotes
 		}
-		val = format(val, { parser, ...this.codeInterpolationOptions });
+		val = format(val, { parser, ...options });
+		logger.debug('what is happening', val);
```

Comment:
:eyes: :laughing:

Reply:
haha I thought I got rid of that my bad.